### PR TITLE
advanced payout doesnt just take percentages now

### PR DESCRIPTION
**Pull request by owocki**
##### Description

allows you to just put in the numbers for advanced payout

##### Refers/Fixes

https://gitcoincore.slack.com/archives/CCL5Q0G5S/p1559100428001200

##### Testing

tested on my local

### CODE REVIEW

File: app/assets/v2/js/pages/bulk_payout.js [hunk 1]
```diff
@@ -23,18 +23,33 @@ $(document).ready(function($) {
...
+  $(document).on('click', '.toggleType', function(event) {
+    event.preventDefault();
+    if ($(this).data('type') == 'no') {
+      $(this).html('(%)');
+      $(this).data('type', 'pct');
+    } else {
+      $(this).html('(#)');
+      $(this).data('type', 'no');
+    }
+    // $(".input_amount").html("0");
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bulk_payout.js]: need to remove this

**Comment by owocki** [comment]
do i press the big scary merge button?  or do u guys do that?

**Comment by danlipert** [comment]
@owocki I'm going to cherry-pick the squashed version of this into the `stable-backup-060619` branch for now, and then stable will be back to normal after this deploy

**Comment by danlipert** [comment]
cherry-picked as 185d9f412

**Pull request closed without merging**